feelslikemonday/app/src/androidTest/java/com/example/feelslikemonday/DAO/FollowRequestDAOTest.java

```diff
@@ -48,6 +58,23 @@ public void onCallback() {
                 signal.countDown();
             }
         });
+
+        DAO.get("xiaole", new FollowRequestCallback() {
+            @Override
+            public void onCallback(FollowRequest followRequest) {
```

Comment:
These tests might not work as intended. For every request we make, the CountDownLatch needs to be set to the same amount (Has to do with how semaphores and multithreading work, if you have questions let me know).